src/scss/06_components/navigation/toggle/_profile-block.scss

```diff
@@ -0,0 +1,186 @@
+.profile {
+  @include font('special.navigation.icon_label');
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: center;
+  width: 100%;
+
+  .hel-icon {
+    --icon-size: 24px;
+    margin-bottom: $spacing-half;
+
+    @include breakpoint($breakpoint-l) {
+      --icon-size: 28px;
+    }
+  }
+}
+
+.profile__login-link {
+  @include font('special.navigation.icon_label');
+  align-items: center;
+  border-left: 1px solid $color-black-20;
+  display: flex;
+  flex-direction: column;
+  grid-area: profile;
+  grid-column-end: -1;
+  justify-content: center;
+  margin-left: 16px;
+  padding-left: 4px;
+  padding-right: 4px;
+  text-decoration: none;
+  text-overflow: ellipsis;
+  width: 65px; // Same as the height of the branding header.
+  z-index: 1;
+
+  @include breakpoint($breakpoint-m) {
+    width: 86px; // Same as the height of the branding header.
+  }
+
+  &::before {
+    @include pseudo-icon('user', 22px, currentColor, block);
+    margin-bottom: $spacing-quarter;
+
+    @include breakpoint($breakpoint-m) {
+      @include pseudo-icon('user', 28px, currentColor, block);
+    }
+  }
+
+  &:hover {
+    text-decoration: underline;
+  }
+
+  &:focus {
+    outline: 3px solid $color-black;
+    outline-offset: -3px;
+  }
+}
+
+.profile__logout-link {
+  @include font('body');
+  border-bottom: 1px solid $color-black-20;
```

Comment:
`border-block` sets both top and bottom borders, autoprefixer will fix it in post processing for older browsers as long as needed.